### ISSUE TITLE
Watching a video in Quicktime over sshfs causes stuck process

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tenei17/macfuse#23]
```
What steps will reproduce the problem?
1. Mount a sshfs drive containing *.m4v video
2. Open video in Quicktime
3. Attempt to close the video

What is the expected output? What do you see instead?
Quicktime is able to close the document window, but unable to exit. Shows
up as a stuck process. The process is flagged uninteruptible, so forcing a
reboot from the command line is neccassary to unmount the file system or
use Quicktime again.

What version of the product are you using? On what operating system?
OSX 10.4.8
macfuse 0.1.0b006 from binaries

Please provide any additional information below.


```

Original issue reported on code.google.com by `white1...@gmail.com` on 13 Jan 2007 at 11:11

**Comment by GoogleCodeExporter** [comment]
```
You can pass '-d' to the sshfs command to get some additional debug information 
that you could post here 
while the above problem occurs. :)
```

Original comment by `chucker...@gmail.com` on 13 Jan 2007 at 1:26

**Comment by GoogleCodeExporter** [comment]
```
Debug log attached
```

Original comment by `white1...@gmail.com` on 13 Jan 2007 at 8:27

Attachments:
- [sshfs.log](https://storage.googleapis.com/google-code-attachments/macfuse/issue-23/comment-2/sshfs.log)

**Comment by GoogleCodeExporter** [comment]
```
Please see if you still have this issue in MacFUSE Core 0.1.9.
```

Original comment by `si...@gmail.com` on 29 Jan 2007 at 3:32

**Comment by GoogleCodeExporter** [comment]
```
Marking this as fixed since 0.1.7 and newer released fixed a bunch of 
termination/hanging issues, and I can't 
reproduce this. Please report back if anybody still has this problem.
```

Original comment by `si...@gmail.com` on 5 Feb 2007 at 8:31
- Changed state: **Fixed**
